src/rust/grapl-config/src/lib.rs

```diff
@@ -65,11 +65,13 @@ pub fn grapl_log_level() -> log::Level {
 }
 
 pub fn _init_grapl_log(service_name: &str) {
-    let filter = EnvFilter::from_default_env().add_directive(
-        format!("{}={}", service_name, grapl_log_level())
-            .parse()
-            .expect("Invalid directive"),
-    );
+    let filter = EnvFilter::from_default_env()
+        .add_directive(format!("warn").parse().expect("Invalid directive"))
```

Comment:
Should we instead set the dependency log level to `GRAPL_LOG_LEVEL` or maybe `GRAPL_DEP_LOG_LEVEL` or something configurable?

Reply:
RUST_LOG (loaded by from_default_env) should override it already, this is just a default basically.

Reply:
Ah got it, thanks